common/decrees/00_decree.txt

```diff
@@ -231,7 +231,7 @@ decree_promote_social_innovation = {
 		academia
 	}
 	modifier = {
-		state_education_access_add = 0.1 # Was 0.25
+		state_education_access_add = 0.08 # Was 0.25
 		state_pop_qualifications_mult = 0.2 # Was 0.2
```

Comment:
you can the remove # was 0.2 if it is also 0.2. Like, you either trust your documentation or you don't xd.